terraform/environments/prd/aws/main.tf

```diff
@@ -30,3 +30,31 @@ module "vpcs" {
   internet_gw_name     = each.value.internet_gw_name
   nat_gws              = each.value.nat_gws
 }
+
+module "alb" {
+  for_each                  = local.vpcs
```

Comment:
lets discuss this a bit. Its kind off strange to create per vpc alb because we may have many albs in one vpc. In my opinion we should set the `alb_vpc_*` vars as local vars too. What do you think?